boot/bootutil/src/image_ec256.c

```diff
@@ -30,6 +30,10 @@
 #ifdef MCUBOOT_USE_TINYCRYPT
 #include "tinycrypt/ecc_dsa.h"
 #endif
+#ifdef MCUBOOT_USE_NRF_CC310_BL
```

Comment:
Why `MCUBOOT_USE_NRF_CC310_BL` instead of `MCUBOOT_USE_CC310`?

Reply:
I was thinking that there are other CC310 implementations out in the wild so `#include "nrf_cc310_glue.h"` is a layer how Nordic Semiconductor has implemented their CC310. In this way say if SiLabs want to implement their CC310 into MCUBoot they could use their own define then add their own glue layer. Now that I think about it the include is not really scalable maybe it should only be `cc310_glue.h` then each vendor implements their own cc310 glue file which is included by CMake instead? I don't know how this inter-opts with other consumers of MCUBoot than zephyr

Reply:
I don't have any experience with MyNewt etc so maybe you could comment something about this.